### ISSUE TITLE
foreach pipe StringIndexOutOfBoundsException

### ISSUE THREAD

**Issue by PierreSchwang** [CloudNetService/CloudNet-v3#325]
When executing the command `/cloudnet players online foreach | "message=hi"` the first time, an exception ocurrs in [DefaultCommandMap#formatArgs](https://github.com/CloudNetService/CloudNet-v3/blob/329c21f03a9e668e1ffc2eb5b575c007c533ba44/cloudnet/src/main/java/de/dytanic/cloudnet/command/DefaultCommandMap.java#L222) and the execution takes several seconds until the message is sent.

Full Stacktrace:
```
[20.09 17:19:09.155] ERROR: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
[20.09 17:19:09.155] ERROR:     at java.lang.AbstractStringBuilder.substring(AbstractStringBuilder.java:935)
[20.09 17:19:09.155] ERROR:     at java.lang.StringBuilder.substring(StringBuilder.java:76)
[20.09 17:19:09.155] ERROR:     at de.dytanic.cloudnet.command.DefaultCommandMap.formatArgs(DefaultCommandMap.java:222)
[20.09 17:19:09.155] ERROR:     at de.dytanic.cloudnet.command.DefaultCommandMap.getCommandFromLine(DefaultCommandMap.java:154)
[20.09 17:19:09.155] ERROR:     at de.dytanic.cloudnet.provider.NodeNodeInfoProvider.getConsoleCommand(NodeNodeInfoProvider.java:133)
[20.09 17:19:09.155] ERROR:     at de.dytanic.cloudnet.network.listener.PacketClientSyncAPIPacketListener.handle0(PacketClientSyncAPIPacketListener.java:65)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.network.listener.PacketClientSyncAPIPacketListener.handle(PacketClientSyncAPIPacketListener.java:23)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.driver.network.protocol.DefaultPacketListenerRegistry.handlePacket(DefaultPacketListenerRegistry.java:133)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.driver.network.netty.NettyNetworkServerHandler.lambda$channelRead0$0(NettyNetworkServerHandler.java:67)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultScheduledTask.call(DefaultScheduledTask.java:143)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultTaskScheduler$Worker.run0(DefaultTaskScheduler.java:308)
[20.09 17:19:09.157] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultTaskScheduler$Worker.run(DefaultTaskScheduler.java:279)
[20.09 17:19:14.158] ERROR: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
[20.09 17:19:14.158] ERROR:     at java.lang.AbstractStringBuilder.substring(AbstractStringBuilder.java:935)
[20.09 17:19:14.158] ERROR:     at java.lang.StringBuilder.substring(StringBuilder.java:76)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.command.DefaultCommandMap.formatArgs(DefaultCommandMap.java:222)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.command.DefaultCommandMap.getCommandFromLine(DefaultCommandMap.java:154)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.command.DefaultCommandMap.tabCompleteCommand(DefaultCommandMap.java:77)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.provider.NodeNodeInfoProvider.getConsoleTabCompleteResults(NodeNodeInfoProvider.java:139)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.network.listener.PacketClientSyncAPIPacketListener.handle0(PacketClientSyncAPIPacketListener.java:78)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.network.listener.PacketClientSyncAPIPacketListener.handle(PacketClientSyncAPIPacketListener.java:23)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.driver.network.protocol.DefaultPacketListenerRegistry.handlePacket(DefaultPacketListenerRegistry.java:133)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.driver.network.netty.NettyNetworkServerHandler.lambda$channelRead0$0(NettyNetworkServerHandler.java:67)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultScheduledTask.call(DefaultScheduledTask.java:143)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultTaskScheduler$Worker.run0(DefaultTaskScheduler.java:308)
[20.09 17:19:14.158] ERROR:     at de.dytanic.cloudnet.common.concurrent.DefaultTaskScheduler$Worker.run(DefaultTaskScheduler.java:279)
```

**Comment by 0utplay** [comment]
Could you specify which version of CloudNet you are using ? (use the "me" Command)

**Comment by PierreSchwang** [comment]
CloudNet Hurricane 3.3.0-RELEASE-521c3bc

**Comment by derklaro** [comment]
Fixed for me in latest 3.4.
